src/components/badges/_badges.scss

```diff
@@ -13,6 +13,7 @@ $include-html: false !default;
   .mint-badge {
     @include component;
     @include fixText($badgeFontSize);
+    overflow: visible; // allow badges align as text
```

Comment:
`inline-block` elements has different behavior in regards of baseline depending on `overflow` value.

This should not be an issue after we have `display: flex` in every hole.